### PR TITLE
Add small note on revocation statement in GCP CloudSQL documentation

### PR DESCRIPTION
**Pull request by vinay-gopalan**

### CODE REVIEW

**Review by Zlaticanin — APPROVED** [review]
LGTM! 👍

File: website/content/docs/secrets/databases/mysql-maria.mdx [hunk 1]
```diff
@@ -217,5 +217,16 @@ GRANT CREATEROLE ON <database>.<object> TO "test-user"@"%";
...
+    ```shell-session
+       $ vault write database/roles/my-role \
+           db_name=my-mysql-database \
+           creation_statements="CREATE USER '{{name}}'@'%' IDENTIFIED BY '{{password}}';GRANT SELECT ON *.* TO '{{name}}'@'%';" \
+           revocation_statements="DROP USER '{{name}}'@'%';" \
+           default_ttl="1h" \
+           max_ttl="24h"
+    ```
+
+Once the role configuration is complete, the steps generate credentials are the same as the ones listed above.
```
Comment (fairclothjm) [inline website/content/docs/secrets/databases/mysql-maria.mdx:232]:
```suggestion
Once the role configuration is complete, the steps to generate credentials are the same as the ones listed above.
```

File: website/content/docs/secrets/databases/mysql-maria.mdx [hunk 2]
```diff
@@ -217,5 +217,16 @@ GRANT CREATEROLE ON <database>.<object> TO "test-user"@"%";
         service_account_json="@my_credentials.json"
     ```
 
-Once the connection has been configured and IAM authentication is complete, the steps to set up a role and generate
-credentials are the same as the ones listed above.
+1.  Configure a role in Vault as done above. Here you can override Vault's default revocation statements to simply
+    drop the user:
```
Comment (schavis) [inline website/content/docs/secrets/databases/mysql-maria.mdx:221]:
```suggestion
1. Configure a new role in Vault but override the default revocation statements
   so Vault will drop the user instead:
```
Style correction: write in declarative statements

File: website/content/docs/secrets/databases/mysql-maria.mdx (hunk 1 above)
Comment (schavis) [inline website/content/docs/secrets/databases/mysql-maria.mdx:232]:
```suggestion
When you finish configuring the new role, generate credentials as before:
```

Style correction: write in declarative statements for instructions.

Comment (schavis) [inline website/content/docs/secrets/databases/mysql-maria.mdx:232]: We should list the actual commands here, not send the reader elsewhere in the doc. If we expect to use the steps under multiple situations, we can always make those steps a partial so it's easier to repeat them.

**Review by schavis — CHANGES_REQUESTED** [review]

File: website/content/docs/secrets/databases/mysql-maria.mdx (hunk 2 above)
Comment (vinay-gopalan) [reply website/content/docs/secrets/databases/mysql-maria.mdx]: Updated, thanks!

**Review by schavis — APPROVED** [review]

**Pull request merged**